src/content/doc-surrealql/clauses/order_by.mdx

```diff
@@ -0,0 +1,49 @@
+---
+sidebar_position: 1
+sidebar_label: ORDER BY
+title: ORDER BY clause | SurrealQL
+description: The `ORDER BY` clause specifies the sort order of the records in a table.
+---
+
+# `ORDER BY` clause
+
+To sort records, SurrealDB allows ordering on multiple fields and nested fields. Use the `ORDER BY` clause to specify a comma-separated list of field names that should be used to order the resulting records. 
+
+The `ASC` and `DESC` keywords can be used to specify whether results should be sorted in an ascending or descending manner. The `COLLATE` keyword can be used to use Unicode collation when ordering text in string values, ensuring that different cases, and different languages are sorted in a consistent manner. Finally, the `NUMERIC` can be used to correctly sort text which contains numeric values.
+
+## Syntax
+
+```syntax title="Clause Syntax"
+[ ORDER [ BY ]
+		@fields [
+			RAND()
+			| COLLATE
```

Comment:
Huh, I didn't even know about COLLATE and NUMERIC! I think these could be turned into real samples like this one which shows the difference between using NUMERIC and not:

```
CREATE person SET num = "1";
CREATE person SET num = "9";
CREATE person SET num = "11";

SELECT * FROM person ORDER BY num NUMERIC;
SELECT * FROM person ORDER BY num NUMERIC;
```